zerver/webhooks/gitlab/tests.py

```diff
@@ -526,10 +527,11 @@ def test_system_merge_request_created_with_custom_topic_in_url(self) -> None:
 
     def test_system_merge_request_created_with_assignee_event_message(self) -> None:
         expected_topic = "my-awesome-project / MR #3 New Merge Request"
-        expected_message = "Tomasz Kolek created [MR #3](https://gitlab.com/tomaszkolek0/my-awesome-project/merge_requests/3) (assigned to Tomasz Kolek) from `tomek` to `master`:\n\n~~~ quote\ndescription of merge request\n~~~"
+        expected_message = "Tomasz Kolek created [MR #3](https://gitlab.com/tomaszkolek0/my-awesome-project/merge_requests/3) (assigned to tomaszkolek0) from `tomek` to `master`:\n\n~~~ quote\ndescription of merge request\n~~~"
```

Comment:
I think we'd ideally be using the name, rather than the ID, given that both seem available in fixtures?  At least, we're using the names for "Tomasz Kolek created ..." at the start, and it seems better to be consistent.

Reply:
Got it 👍🏽